httputil/middleware.go

```diff
@@ -52,8 +52,11 @@ func HTTPDebugMiddleware(out io.Writer, printBody bool, logger func(...interface
 
 			next.ServeHTTP(recorder, r)
 
+			for k, v := range recorder.Header() {
+				w.Header()[k] = v
```

Comment:
Change this to https://golang.org/pkg/net/http/#Header.Set
```w.Header().Set(k,v)```

Reply:
It's a bit more complicated because v is []string. But i can do it explicitly.